packages/experimental/RadioGroup/src/Radio/Radio.styling.ts

```diff
@@ -56,15 +57,38 @@ export const stylingSettings: UseStylingOptions<RadioProps, RadioSlotProps, Radi
       }),
       ['radioInnerCircleSize', 'radioVisibility', 'radioFill'],
     ),
+    content: buildProps(
+      () => ({
+        style: {
+          alignItems: 'flex-start',
+          flexDirection: 'column',
+        },
+      }),
+      [],
+    ),
     label: buildProps(
-      (tokens: RadioTokens) => ({
+      (tokens: RadioTokens, theme: Theme) => ({
         variant: tokens.variant,
         style: {
           marginTop: 2,
+          marginRight: tokens.marginRight,
+          color: tokens.color,
+          ...fontStyles.from(tokens, theme),
+        },
+      }),
+      ['color', 'marginRight', ...fontStyles.keys],
```

Comment:
variant should be added here as a cache key

Reply:
Added, thanks!